charms/katib-db-manager/src/charm.py

```diff
@@ -23,6 +23,7 @@
 ]
 MYSQL_WARNING = "Relation mysql is deprecated."
 UNBLOCK_MESSAGE = "Remove deprecated mysql relation to unblock."
+SERVICE_PORT = "6789"
```

Comment:
comment with where it is hardcoded in upstream?

Reply:
can you link the 0.16 version instead?

Reply:
added